common/pom.xml

```diff
@@ -128,5 +128,19 @@
             <artifactId>jutf7</artifactId>
             <version>1.0.0</version>
         </dependency>
+
+        <!-- test -->
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <version>1.17.6</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.hubspot.jackson</groupId>
+            <artifactId>jackson-datatype-protobuf</artifactId>
+            <version>0.9.12</version>
+        </dependency>
```

Comment:
hi We are trying our best to reduce the number of third-party dependency packages. It is not recommended to introduce more third-party packages for testing.

Reply:
got it.